### PR TITLE
add gclient high load tests

### PR DESCRIPTION
**Pull request by MikitaS**
add async and non-async high load testing

### CODE REVIEW

**Comment by MikitaS** [comment]
@LouiseMedova PTAL

File: ft-logic/src/lib.rs [hunk 1]
```diff
@@ -316,25 +316,45 @@ impl FTLogic {
...
-    let logic: &mut FTLogic = unsafe { FT_LOGIC.get_or_insert(Default::default()) };
-    match action {
-        FTLogicAction::Message {
-            transaction_hash,
-            account,
-            payload,
-        } => logic.message(transaction_hash, &account, &payload).await,
-        FTLogicAction::UpdateStorageCodeHash(storage_code_hash) => {
-            logic.update_storage_hash(storage_code_hash)
+fn __main_safe() {
```
Comment (LouiseMedova) [inline ft-logic/src/lib.rs:319]: Change to `async fn main`

File: ft-logic/src/lib.rs [hunk 2]
```diff
@@ -316,25 +316,45 @@ impl FTLogic {
...
+#[no_mangle]
+unsafe extern "C" fn handle_reply() {
+    gstd::record_reply();
+}
+#[no_mangle]
+unsafe extern "C" fn handle() {
+    __main_safe();
+}
+#[no_mangle]
+unsafe extern "C" fn handle_signal() {
```
Comment (LouiseMedova) [inline ft-logic/src/lib.rs:352]: Remove

File: ft-main/src/lib.rs [hunk 3]
```diff
@@ -150,6 +153,18 @@ async fn main() {
 
 #[no_mangle]
 unsafe extern "C" fn init() {
+    debug!("INIT");
```
Comment (LouiseMedova) [inline ft-main/src/lib.rs:156]: Remove all debugs

**Review by LouiseMedova — CHANGES_REQUESTED** [review]

File: ft-logic/src/lib.rs (hunk 1 above)
Comment (MikitaS) [reply ft-logic/src/lib.rs:319]: it's being called from extern "C" so how it's possible to be async?

File: ft-logic/src/lib.rs (hunk 2 above)
Comment (MikitaS) [reply ft-logic/src/lib.rs:352]: done

File: ft-main/src/lib.rs (hunk 3 above)
Comment (MikitaS) [reply ft-main/src/lib.rs:156]: done

**Comment by shamilsan** [comment]
Stale PR? @MikitaS

**Pull request closed without merging**